uxarray/cross_sections/dataarray_accessor.py

```diff
@@ -32,9 +32,11 @@ def constant_latitude(self, lat: float, method="fast"):
         method : str, optional
             The internal method to use when identifying faces at the constant latitude.
             Options are:
-            - 'fast': Uses a faster but potentially less accurate method for face identification.
-            - 'accurate': Uses a slower but more accurate method.
-            Default is 'fast'.
+            - 'edge_intersection': The intersection of each edge with a line of constant latitude is calculated, with
+            faces that contain that edges included in the result.
+            - 'bounding_box_intersection': The minimum and maximum latitude of each face is used to determine if
```

Comment:
please gives a better name, the 'bounding_box_intersection' is not appropriate here.

Bounding box is a process to describe how to select the candidate faces.

And how we calculate the intersection is another things.

Perhaps we need two param: candidate_faces_selection and intersection_caluclation

Reply:
What about having a `use_bounding_box=True/False` parameter, which could be used to toggle between the accurate implementation written around the bounding box and the currently faster one that I wrote?

Reply:
Sounds good